nucleus/flashlight/framework/src/main/java/org/glassfish/flashlight/provider/FlashlightProbe.java

```diff
@@ -264,7 +264,7 @@ public String getModuleProviderName() {
 	}
 
     public String getModuleName() {
-        return moduleProviderName;
```

Comment:
Looking at the class I am pretty sure this was actually a case of maybe auto-completing to the wrong field. There is another getter for the `moduleProviderName` and this should use the `moduleName`.

Reply:
I don't know any details on flashlight but I was thinking that maybe the `getProbeDesc` is used as or for the key when I was looking why such a field confusion would go unnoticed for years. At least a method called `createProbe` uses it to do:
```java
FlashlightProbe fp = genericProvider.getProbe(probeDescriptor);
```

Reply:
Yes, I wasn't able to trigger code that would involve this method both over HTTP or JMX interfaces, so it's likely safe to remove it.